.github/workflows/release.yml

```diff
@@ -8,15 +8,17 @@ jobs:
 
     steps:
       - name: Checkout source
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Need the full history so that `setuptools-scm` can calculate the version correctly.